tests/core/test_planetary_nodes.py

```
@@ -124,3 +148,38 @@ def test_jupiter_ascending_node_matches_swe(self):
+        )
+        tropical = AstrologicalSubjectFactory.from_birth_data("Nodes Tropical", **birth)
+        sidereal = AstrologicalSubjectFactory.from_birth_data(
+            "Nodes Lahiri", **birth, zodiac_type="Sidereal", sidereal_mode="LAHIRI"
+        )
+        assert sidereal.ayanamsa_value is not None
+        ayanamsa = sidereal.ayanamsa_value
+
+        trop_nodes = PlanetaryNodesFactory.from_subject(tropical, planets=["Mars", "Jupiter"])
+        sid_nodes = PlanetaryNodesFactory.from_subject(sidereal, planets=["Mars", "Jupiter"])
+
+        for trop, sid in zip(trop_nodes.nodes, sid_nodes.nodes):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)